Rubberduck.Core/UI/GridViewSort.cs

```diff
@@ -1,40 +1,40 @@
-﻿using System.Collections.Generic;
-using System.Linq;
+﻿//using System.Collections.Generic;
```

Comment:
Did you mean to leave this commented-out file in?

Reply:
Nope, I commented it out to see if it was used and forgot to remove it.  Will fix.